internal/container/controller.go

```diff
@@ -231,6 +233,40 @@ func (c Controller) ContainerFiles(id string, filespec string) (files []string)
 	return strings.Split(string(stdout), "\n")
 }
 
+func (c Controller) CopyFile(id string, src string, destFolder string) {
+	if id == "" {
+		panic("Must pass in non-empty id")
+	}
+	if src == "" {
+		panic("Must pass in non-empty src")
+	}
+	if destFolder == "" {
+		panic("Must pass in non-empty destFolder")
+	}
+
+	_, f := filepath.Split(src)
+	h, err := os.ReadFile(src)
+	checkErr(err)
+
+	// Create and add some files to the archive.
+	var buf bytes.Buffer
+	tw := tar.NewWriter(&buf)
```

Comment:
>tw := tar.NewWriter(&buf) [](http://example.com/codeflow?start=1&length=25)

instead of having tw.Close below, shouldn't there be a "defer tw.Close()" call right after this? Nobody ever error checks a Close afaik